Bottom: 8b8dee939b3afce1eecd6cd8ad606566ee872f1c
Top:    5a719bf8cd42783e6dee9570082727635eda62b7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-13 19:18:26 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
